documentation/tutorials/check.md

```diff
@@ -99,6 +99,39 @@ We can then run `check` with the `-drift` flag:
 
 This creates another pair of HTML and JSON reports. This time the files contain a drift report, showing how the target database schema (in this case `foobar`) differs from whats contained in the applied migrations.
 
+### Programmatically identifying drift
+<h4> <i class="fa fa-windows"></i> Windows </h4>
+
+Powershell script to parse the output JSON,find the last drift report it contains and report the status of the flag
+
+<pre class="console">
+$ip = Get-Content "check_report.json" -Raw | convertfrom-json
+# Extract all the objects that are drift reports
+$drifts = $ip.individualResults | where-Object operation -eq "drift"
+# check if there is anything in the list and look at the status of the last item in the list (most recent drift report)
+if ( ($null -ne $drifts ) -and ($drifts[-1].driftDetected -eq "True") )
+{
+    Write-Output "Drift detected"
+    exit 1
+} else {
+    exit 0
+}</pre>
+
+<h4> <i class="fa fa-linux"></i> Linux </h4>
+
+Bash script to parse the output JSON - this depends on [jq]("https://stedolan.github.io/jq/")
```

Comment:
The link doesn't seem to work when I try generating the docs:

![image](https://user-images.githubusercontent.com/60435811/196375331-c3f209b5-f1b1-4f04-98cb-109edff59d71.png)

Might need to remove the quotes?

Reply:
That's the one - now sorted